src/organizercore.h

```diff
@@ -211,6 +212,20 @@ class OrganizerCore : public QObject, public MOBase::IPluginDiagnose
     friend class OrganizerCore;
   };
 
+  // enumeration for the groups where refresh callbacks can be put
+  //
+  enum class RefreshCallbackGroup : int
+  {
+    // for callbacks by the core itself, highest priority
+    CORE = 0,
+
+    // internal MO2 callbacks
+    INTERNAL = 1,
+
+    // plugins callbacks
```

Comment:
I know this is MO2 extension plugins, but it might be confusing for new devs if this is used in stuff like `SavePluginList` or `RefreshESPList`. 

Comment should make this clearer

Reply:
I've rename the enum entry to `EXTERNAL` and change the comment.